packages/tideline/plugins/blip/basics/components/CalendarContainer.js

```diff
@@ -129,6 +129,9 @@ class CalendarContainer extends React.Component {
     // Could be subject to change so I thought this was preferred over
     // hard-coding a solution that assumes Monday is the first day
     // of the week.
+    if (this.props.days.length === 0) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why is this necessary? (also, the comment belongs to the code below what you added)

Reply:
It seems that some patient could get an application crash because of the line:
```
var firstDay = moment.utc(this.props.days[0].date).day()
```
`this.props.days` can now be an empty array, meaning that `this.props.days[0]` can be undefined and `this.props.days[0].date` throws an NPE